libs/design-system/src/config/vars.ts

```diff
@@ -0,0 +1,74 @@
+import { getContextPath, NovuComponentEnum } from '@novu/shared';
```

Comment:
Same as with the HTTP client I struggle to see environment variables management in a Design System. Reasoning would be similar. But I am open to hear the goal of this.

Reply:
My comment above touches this as well.